Pilot/pilotTools.py

```diff
@@ -214,53 +214,97 @@ def listdir(directory):
     return contents
 
 
-def getFlavour(ceName):
+def getSubmitterInfo(ceName):
+    """
+    Check the environment variables to determine the type of batch system and CE used 
+    to submit the pilot being used and return this information in a tuple.
+    """
 
-    pilotReference = os.environ.get("DIRAC_PILOT_STAMP", "")
-    flavour = "DIRAC"
+    pilotReference = os.environ.get("DIRAC_PILOT_STAMP", "Unknown")
+    # Batch system taking care of the pilot
+    # Might be useful to extract the info to interact with it later on
+    batchSystemType = None
+    batchSystemJobID = None
+    batchSystemParameters = {
+        "BinaryPath": "Unknown",
+        "Host": "Unknown",
+        "InfoPath": "Unknown",
+        "Queue": "Unknown",
+    }
+    # Flavour of the pilot
+    # Inform whether the pilot was sent through SSH+batch system or a CE
+    flavour = "Unknown"
 
     # # Batch systems
 
-    # Take the reference from the Torque batch system
+    # Torque
     if "PBS_JOBID" in os.environ:
-        flavour = "SSHTorque"
-        pilotReference = "sshtorque://" + ceName + "/" + os.environ["PBS_JOBID"].split(".")[0]
+        batchSystemType = "Torque"
+        batchSystemJobID = os.environ["PBS_JOBID"]
+        batchSystemParameters["BinaryPath"] = os.environ.get("PBS_O_PATH", "Unknown")
+        batchSystemParameters["Queue"] = os.environ.get("PBS_O_QUEUE", "Unknown")
+
+        flavour = "SSH%s" % batchSystemType
+        pilotReference = "sshtorque://" + ceName + "/" + batchSystemJobID.split(".")[0]
 
-    # Take the reference from the OAR batch system
+    # OAR
     if "OAR_JOBID" in os.environ:
-        flavour = "SSHOAR"
-        pilotReference = "sshoar://" + ceName + "/" + os.environ["OAR_JOBID"]
+        batchSystemType = "OAR"
+        batchSystemJobID = os.environ["OAR_JOBID"]
+
```

Comment:
`batchSystemParameters["BinaryPath"]` and ` batchSystemParameters["Queue"]` for this case?

Reply:
Mmm maybe @atsareg knows what was this for. The feeling is that it could be dropped...

Reply:
From what I have seen, the project is not very known but still exists and has active developers working on it.